src/bin/pg_autoctl/monitor.c

```diff
@@ -1727,15 +1728,17 @@ monitor_ensure_extension_version(Monitor *monitor,
 								 MonitorExtensionVersion *version)
 {
 	const char *extensionVersion = PG_AUTOCTL_EXTENSION_VERSION;
+	char envExtensionVersion[MAXPGPATH];
 
-	/* in test environement, we can export any target version we want */
-	if (getenv(PG_AUTOCTL_DEBUG) != NULL)
+	/* in test environment, we can export any target version we want */
+	if (get_env_variable(PG_AUTOCTL_DEBUG, NULL, 0) >= 0L)
```

Comment:
It might be useful to have a function that just checks if the env is set or not, e.g:
```c
bool is_env_variable_set(char *name)
```

Seems like a common case.

Reply:
The EnvVar struct could have a `bool found` slot to keep the information, and this function would then return the value from the slot when it's been initialised. So another slot `bool initialised` would be good too I guess.